src/test/resources/projects/react/reference/react-app/public/index.html

```diff
@@ -3,8 +3,23 @@
   <head>
     <title>React App</title>
     <meta charset="utf-8" />
-    <meta property="cq:pagemodel_root_url" content="/content/test-spa-react-project/en.model.json"/>
-    <link rel="stylesheet" href="http://localhost:4502/etc.clientlibs/test-spa-react-project/clientlibs/responsivegrid.css" type="text/css">
+    <meta
+      property="cq:pagemodel_root_url"
+      content="/content/test-spa-react-project/en.model.json"
+    />
+    <link
+      rel="stylesheet"
+      href="http://localhost:4502/etc.clientlibs/test-spa-react-project/clientlibs/responsivegrid.css"
+      type="text/css"
+    />
+    <script>
+      // On localhost (dev server), redirect from "/" to app root
+      const isLocalhost =
```

Comment:
I think a cleaner solution is to check if we are in dev mode rather then check for localhost/127.0.0.1. 

For example:
`if (location.pathname == '/' && '%NODE_ENV%' === 'development' && '%REACT_APP_ROOT%') { 
  location.href = '%REACT_APP_ROOT%';
}`

Reply:
Good idea, I'll update the code